pysat/instruments/iss_fpmu.py

```diff
@@ -75,7 +76,7 @@ def clean(inst):
 
     Notes
     --------
-    No cleaning currently available for FPMU
+
```

Comment:
We should probably let people know it only replaces fill values with a NaN

Reply:
Agreed.  Fixed.